### PR TITLE
HDDS-3731. [doc]add storage space quota document.

### PR DESCRIPTION
**Pull request by captainzmc**
## What changes were proposed in this pull request?

The space Quota feature is now nearly complete, so add a description of how to use it

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-3731

## How was this patch tested?
no need test

### CODE REVIEW

File: hadoop-hdds/docs/content/feature/Quota.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+So far, we know that Ozone allows users to create volumes, buckets, and keys. A Volume usually contains several buckets, and each Bucket also contains a certain number of keys. Obviously, it should allow the user to define quotas (for example, how many buckets can be created under a Volume or how much space can be used by a Bucket), which is a common requirement for storage systems.
+
+## Currently supported
+1. Storage Space level quota
+
+Administrators should be able to define how much storage space a Volume or Bucket can use.
+
+## Client usage
+### Storage Space level quota
+Storage space level quotas allow the use of units such as KB (k), MB (m), GB (g), TB (t), PB (p), etc. Represents how many storage Spaces will be used.
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/feature/Quota.md:38]: how many => how much

File: hadoop-hdds/docs/content/feature/Quota.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+So far, we know that Ozone allows users to create volumes, buckets, and keys. A Volume usually contains several buckets, and each Bucket also contains a certain number of keys. Obviously, it should allow the user to define quotas (for example, how many buckets can be created under a Volume or how much space can be used by a Bucket), which is a common requirement for storage systems.
+
+## Currently supported
+1. Storage Space level quota
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/feature/Quota.md:32]: Do we have a section for name space quota?

File: hadoop-hdds/docs/content/feature/Quota.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+bin/ozone sh bucket create --space-quota 5m /volume1/bucket1
+```
+That means bucket1 allows us to use 5MB of storage.
+
+```shell
+bin/ozone sh bucket setquota  --space-quota 10g /volume1/bucket1 
+```
+This behavior changes the quota for Bucket1 to 10GB
+
+A bucket quota should not be greater than its Volume quota. Let's look at an example. If we have a 10MB Volume and create five buckets under that Volume with a quota of 5MB, the total quota is 25MB. In this case, the bucket creation will always succeed, and we check the quota for bucket and volume when the data is actually written. Each write needs to check whether the current bucket is exceeding the limit and the current total volume usage is exceeding the limit.
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/feature/Quota.md:62]: what if the case of volume quota = 10MB but create a bucket of quota 20MB. Will the bucket create fail?

File: hadoop-hdds/docs/content/feature/Quota.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+#### Clear the quota for Volume1. The Bucket cleanup command is similar.
+```shell
+bin/ozone sh volume clrquota --space-quota /volume1
+```
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/feature/Quota.md:67]: Can you add a section on how to check the quota and usage?

**Review by xiaoyuyao — COMMENTED** [review]
LGTM, some minor comments added inline.

File: hadoop-hdds/docs/content/feature/Quota.md (hunk 2 above)
Comment (captainzmc) [reply hadoop-hdds/docs/content/feature/Quota.md:32]: The name space Quota feature is currently not available and is under development. We can refine the document after it is complete.

File: hadoop-hdds/docs/content/feature/Quota.md (hunk 3 above)
Comment (captainzmc) [reply hadoop-hdds/docs/content/feature/Quota.md:62]: Yes, the quota value of bucket cannot be greater than the quota value of volume

**Comment by captainzmc** [comment]
Thanks @xiaoyuyao 's review.  I have submitted a new commit fix the above issues.

**Comment by xiaoyuyao** [comment]
Thanks @captainzmc  for the update. LGTM, +1.

**Pull request merged**